stan/math/opencl/kernels/divide_columns.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef STAN_MATH_OPENCL_KERNELS_DIVIDE_COLUMNS_HPP
+#define STAN_MATH_OPENCL_KERNELS_DIVIDE_COLUMNS_HPP
+#ifdef STAN_OPENCL
+
+#include <stan/math/opencl/kernel_cl.hpp>
+#include <stan/math/opencl/buffer_types.hpp>
+
+namespace stan {
+namespace math {
+namespace opencl_kernels {
+// \cond
+static const char *divide_columns_kernel_code = STRINGIFY(
+    // \endcond
+    /**
+     * Takes vector A and divides columns vector in A element-wise by the values
+     * in vec
+     * @param[out] A  Matrix to be divided elementwise
+     * @param[in] vec  Vector to divide A by
+     * @param vec_size Size of elementwise divisor.
+     * @note Code is a <code>const char*</code> held in
+     * <code>divide_columns_vec_kernel_code.</code>
+     */
+    __kernel void divide_columns_vec(__global double *A, __global double *vec,
+                                     int vec_size) {
+      const int i = get_global_id(0);
+      A[i] /= vec[i % vec_size];
+    }
+    // \cond
+);
+// \endcond
+/**
+ * See the docs for \link kernels/divide_columns.hpp divide_columns_vec()
+ * \endlink
+ */
+const kernel_cl<out_buffer, in_buffer, int> divide_columns_vec(
+    "divide_columns_vec", {indexing_helpers, divide_columns_kernel_code});
+
+// \cond
+static const char *divide_column_scalar_kernel_code = STRINGIFY(
+    // \endcond
+    /**
+     * Performs element-wise division on @c A
```

Comment:
what's `@c`?

Reply:
Found this out from @yizhang-yiz 

> \c <word>
> Displays the argument using a typewriter font. Use this to refer to a word of code. Equivalent to <tt>word</tt>.

https://stackoverflow.com/a/25686642

Reply:
Rad. It’s good we use features of the tools we have, especially if they’re clear and in the doc. A request: Could you go from using the at symbol to backslash? There’s no reason for us to use both within the same documentation block.

Reply:
Rad. It’s good we use features of the tools we have, especially if they’re clear and in the doc. A request: Could you go from using the at symbol to backslash? There’s no reason for us to use both within the same documentation block.

Reply:
Sure!